File: sum-birds/sum-birds.go
func sumBirds(number1 int, number2 int, number3 int)int{
    array:= []int{number1,number2,number3}
    /* write your code here */
	sum := 0;

    for _,v := range array {
        sum += v;
    }
    return sum;
}

File: sum-birds/sum-birds.js
function sumBirds(number1, number2, number3) {
    const array = [number1, number2, number3];
    /* write your code here */
    let sum = 0;

    for (var i = 0; i < array.length; i++) {
        sum += array[i];
    }
    return sum;
}
